core/dbt/task/base.py

```diff
@@ -286,6 +287,13 @@ def skip_result(self, node, message):
     def compile_and_execute(self, manifest, ctx):
         result = None
         with self.adapter.connection_for(self.node):
+            ctx.node._event_status['node_status'] = str(RunningStatus.Compiling)
```

Comment:
is this string representation of this class name user visible? the conversion to user-visible strings should live in the events module if possible.

Reply:
The problem is that the node_status can contain either RunningStatus or NodeStatus, so it's just more work to get that straight.

Reply:
it is more work, but I don't want us to change a classname and break an app parsing our logs.

Reply:
I'm not sure what you're asking about here. When enums like NodeStatus are serialized, I think they're always strings.